openid-4-verifiable-presentations-1_0.md

```diff
@@ -1552,6 +1552,7 @@ try {
         protocol: "openid4vp",
         request:  {
           response_type: "vp_token",
+          response_mode: "is_w3c_dc_api",
```

Comment:
Perhaps I am missing something but where did the `is_` come from? 
```suggestion
          response_mode: "w3c_dc_api",
```

Reply:
The only thing you're missing is a careless mistake on my part when copying content for the several different places one has to write something for a PR... Thanks for catching it!